### ISSUE TITLE
instructions

### ISSUE THREAD

**Issue by tloudon** [pdxruby/bot#5]
@aarontc,

Once we have the basic architecture setup and can post to twitter, it would be good to write down the workflow in the `README.md`.  My basic idea was someone writes some announcement content in a flat file of some kind and then scps it to the server.  So we probably need to let folks know how to get access to the server and what the various flat files or whatever are--basic usage, etc.

Thanks!

//cc @danascheider

**Comment by aarontc** [comment]
@tloudon I envision the basic set up would include a relational database (Postgres is my preference) to store notification templates per medium (twitter, email, slack, calagator, ...?), and scheduled action rules (along with a gem like PerfectSched to manage invocation.) I would be ok with a CLI to start with, but a (very) minimalist web interface might be a better long-term approach so scheduling notifications and editing templates can be accomplished by multiple users, without too much SSH/SCP reliance.

Unless you object, I'd choose docker for production builds, and use Jenkins to manage deploying those images to whatever machine we end up hosting the service on.

Documenting all these pieces in README.md would be essential :)

**Comment by tloudon** [comment]
@aarontc, I agree a web gui makes more sense over scp (and whatever db is fine w/ me).  I just don't want to overburden you w/ an ORM, auth, etc straight outta the gate.  You know your schedule and availability best, so let's move forward how you want here including docker and jenkins.  @danascheider said she might be able to pitch in some, as did @accua.

Thanks!
